### ISSUE TITLE
Error in _group_psms with bigger_scores_better param

### ISSUE THREAD

**Issue by MKoesters** [ursgal/ursgal#74]
When setting the param 'bigger_scores_better' and not setting 'validation_score_field' we get this error

```
bigger_scores_better = self.UNODE_UPARAMS['bigger_scores_better']['uvalue_style_translation'][search_engine]
UnboundLocalError: local variable 'search_engine' referenced before assignment
```

This is due to the fact, that search_engine is only set, when validation_score_field is None
I guess we should require, that both of these parameters need to be set

**Comment by StSchulze** [comment]
I think validation_score_field does not always require bigger_scores_better, but the other way round, yes.

So I agree, we should check this in the corresponding wrappers to get a clear error message.

**Comment by StSchulze** [comment]
should be fixed with #88
